integreat_cms/cms/templates/analytics/_page_hix_widget.html

```diff
@@ -8,32 +8,51 @@
 {% endblock collapsible_box_title %}
 {% block collapsible_box_content %}
     <div class="flex flex-wrap justify-between gap-2">
-        <p>
+        <p id="hix-list-n-pages">
             {% blocktranslate trimmed with n=page_translations|length %}
                 The list shows the first {{ n }} pages with the lowest HIX value in ascending order.
             {% endblocktranslate %}
         </p>
+        <p id="hix-list-all-pages" class="hidden">
+            {% translate "The list shows all pages with the lowest HIX value in ascending order." %}
+        </p>
         <p>
             <b>{{ ready_for_mt }}%</b> {% translate " of all pages are ready for automatic translation." %}
         </p>
     </div>
     <div class="mt-4">
-        <table id="page_hix_list" class="w-full max-w-xl mb-4">
-            <thead>
-                <tr class="border-b border-gray-400">
-                    <th class="text-sm text-left uppercase p-1">{% translate "Page name" %}</th>
-                    <th class="text-sm text-left uppercase p-1">{% translate "HIX value" %}</th>
-                </tr>
-            </thead>
-            <tbody>
-                {% for page_translation in page_translations %}
-                    {% include "analytics/_page_hix_row.html" %}
-                {% empty %}
-                    {% translate "There is no page." %}
-                {% endfor %}
-            </tbody>
-        </table>
+        {% if page_translations %}
+            <table id="page_hix_list" class="w-full max-w-xl mb-4">
+                <thead>
+                    <tr class="border-b border-gray-400">
+                        <th class="text-sm text-left uppercase p-1">{% translate "Page name" %}</th>
+                        <th class="text-sm text-left uppercase p-1">{% translate "HIX value" %}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for page_translation in page_translations %}
+                        {% include "analytics/_page_hix_row.html" %}
+                    {% endfor %}
+                </tbody>
+                <tbody id="toggle-hix-score-list" class="hidden">
+                    {% for page_translation in toggleable_page_translations %}
+                        {% include "analytics/_page_hix_row.html" %}
+                    {% endfor %}
+                </tbody>
+            </table>
+        {% else %}
+            <p class="my-2 bg-yellow-100 border-l-4 border-yellow-500 text-yellow-600 px-4 py-3 mb-5"
+               role="alert">
+                {% translate "Currently there is not yet a page for which the HIX value has been calculated." %}
```

Comment:
In my opinion, it could make more sense to just not show the widget when no page with HIX value exists?